src/frontend/src/scheduler/plan_fragmenter.rs

```diff
@@ -947,7 +947,7 @@ impl BatchPlanFragmenter {
                 .get_fragment_mapping(&table_catalog.fragment_id)
                 .ok_or_else(|| {
                     anyhow!(
-                        "failed to get the vnode mapping for the `Materialize` of {}",
+                        "failed to get the vnode mapping for the `BatchLookupJoin` of {}",
```

Comment:
I guess we were referring to the vnode mapping of `Materialize` fragment of the base table here. 🤔